### PR TITLE
Adding 3 functions, 6 unit testing and another changes for ShouldAllBeEqual

### PR DESCRIPTION
**Pull request by Haneen96**

### CODE REVIEW

File: src/Shouldly/Configuration/DiffToolConfiguration.cs [hunk 1]
```diff
@@ -54,7 +54,7 @@ public DiffTool GetDiffTool()
             var diffTool = _diffToolPriority.FirstOrDefault(d => d.Exists());
             if (diffTool == null)
                 diffTool = _diffTools.FirstOrDefault(d => d.Exists());
-            
+         //   new[] { 2, 4, 6 }.ShouldBe(new[] { 2,4,8});
```
Comment (sungam3r) [inline src/Shouldly/Configuration/DiffToolConfiguration.cs]: ?

File: src/Shouldly/ShouldlyExtensionMethods/ShouldBeEnumerableTestExtensions.cs [hunk 2]
```diff
@@ -226,14 +229,37 @@ public static void ShouldBeUnique<T>(this IEnumerable<T> actual, string customMe
         {
             ShouldBeUnique(actual, () => customMessage);
         }
-
+
```
Comment (sungam3r) [inline src/Shouldly/ShouldlyExtensionMethods/ShouldBeEnumerableTestExtensions.cs]:
```suggestion

```

File: src/ConsoleApp1/ConsoleApp1.csproj [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Shouldly\Shouldly.csproj" />
+  </ItemGroup>
+
+</Project>
```
Comment (sungam3r) [inline src/ConsoleApp1/ConsoleApp1.csproj]: Does this project need to be merged?

File: src/Shouldly/ShouldlyExtensionMethods/ShouldBeEnumerableTestExtensions.cs [hunk 4]
```diff
@@ -321,6 +347,33 @@ private static List<object> GetDuplicates<T>(IEnumerable<T> items)
...
+        //    var list = new List<object>();
+        //    var ActualGroupedEnumerable = new List<object>();
+
+        //    foreach (object o1 in items)
+        //    {
+        //        ActualGroupedEnumerable.AddRange(list.Where(o2 => o1 != null && o1.Equals(o2)));
+        //        list.Add(o1);
+        //    }
+        //    return ActualGroupedEnumerable;
+        //}
```
Comment (sungam3r) [inline src/Shouldly/ShouldlyExtensionMethods/ShouldBeEnumerableTestExtensions.cs]: ?

**Comment by SimonCropp** [comment]
closing for now as this seems abandoned

**Pull request closed without merging**